docs/internal/maintenance/ReleaseProcess-release.md

```diff
@@ -16,6 +16,7 @@ Release process - Release
 * [11. Produce distribution zip file](#11-produce-distribution-zip-file)
 * [12. Update Zonemaster repository main _README.md_](#12-update-zonemaster-repository-main-readmemd)
 * [13. Generate documents](#13-generate-documents)
+* [13. Generate documents and check public documents](#13-generate-documents-and-check-public-documents)
```

Comment:
This new line is supposed to replace the previous one, isn’t it?

Reply:
Yes. Fixed.